Allow for any install location

justin8 opened the pull request:
Fix for #519

edvinasme commented on install.sh:
```diff
@@ -36,18 +36,20 @@ test -w "$HOME/$CONFIG_FILE" &&
   file_type=$1
   file_to_enable=$2
-  [ ! -d "$BASH_IT/$file_type/enabled" ] && mkdir "$BASH_IT/${file_type}/enabled"
+  mkdir -p "$BASH_IT/${file_type}/enabled"
```
I belive this should be replaced with:

`[ ! -d "$BASH_IT/$file_type/enabled" ] && mkdir -p "$BASH_IT/${file_type}/enabled"`

This will make sure `mkdir` is NOT run if directory already exists.

justin8 replied:
-p won't touch directories if they exist already, but will create the entire path, so it's just longer with no real benefit.

-p was also to create the tree if it didn't exist already

nwinkler commented on install.sh:
```diff
@@ -36,18 +36,20 @@ test -w "$HOME/$CONFIG_FILE" &&
-cp "$HOME/.bash_it/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+cp "$BASH_IT/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+sed -i "s|{{BASH_IT}}|$BASH_IT|" "$HOME/$CONFIG_FILE"
+exit
```
This probably shouldn't be here, right? It terminates the install script a bit too soon :smile:

justin8 replied:
Haha, good catch. I had that in while testing out the sed; I'll patch it tonight

nwinkler commented:
One more thing: It would be great if the documentation would reflect this new feature as well. Maybe you can add a paragraph to the `Install` section explaining that any other location should also work?

justin8 commented:
Will do!

justin8 commented:
So I updated the readme slightly. It didn't explicitly say before that it had to be installed in any particular place, and I do like the idea of giving simple to follow instructions for users (like cloning to a hidden folder in home by default for example), but I made it explicitly mention it can be installed to any location now.

Also fixed that stray exit.

nwinkler commented:
Excellent - thanks!
